### PR TITLE
Keycloak dashboard documentation

### PR DESCRIPTION
**Pull request by psturc**
https://issues.jboss.org/browse/AEROGEAR-1939

### CODE REVIEW

File: services/keycloak-dashboard.adoc [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+After the Metrics service (includes Grafana tool and Prometheus monitoring system) and Keycloak service are provisioned, 
+you should be able to see the "Keycloak Metrics" in the list of available dashboards 
+(navigate to Grafana's exposed URL -> Log in -> Home -> Select *Keycloak Metrics*)
+
+image:img/keycloak-dashboard/keycloak-dashboard-overview.png[keycloak-dashboard]
+
+== Dashboard's panels description
+
+Keycloak dasboard consists of several panels which give you overview of the specific
```
Comment (StevenTobin) [inline services/keycloak-dashboard.adoc]: dasboard -> dashboard

Comment (psturc) [reply services/keycloak-dashboard.adoc]: Thanks Steve. Fixed.

File: services/keycloak-dashboard.adoc [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+* OpenShift Ansible Broker
+** https://docs.openshift.org/latest/install_config/install/advanced_install.html#configuring-openshift-ansible-broker
+* Metrics service deployment from the Service Catalog
+** https://github.com/aerogearcatalog/metrics-apb
+* Keycloak service deployment from the Service Catalog
+** https://github.com/aerogearcatalog/keycloak-apb
+
+*Note*
+
+Metrics service and Keycloak service must be provisioned within the same project,
```
Comment (pb82) [inline services/keycloak-dashboard.adoc]: project is Openshift specific, should we use `namespace` instead?

File: services/keycloak-dashboard.adoc [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+otherwise the Keycloak's metrics endpoint won't reachable for Prometheus and no data will be displayed
```
Comment (pb82) [inline services/keycloak-dashboard.adoc]: `won't reachable` -> `won't be reachable`

File: services/keycloak-dashboard.adoc [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+=== link:http://docs.grafana.org/features/panels/singlestat/#singlestat-panel[Singlestat Panels]
+
+==== Total Registrations
+
+Total number of registered (non-admin) users. This number comprises all successful registrations made via various providers, e.g. Keycloak, Github, Facebook etc.
+
+==== Total Logins
+
+Total number of successful logins (only non-admin users). Also includes logins via various providers.
```
Comment (pb82) [inline services/keycloak-dashboard.adoc]: Maybe better: ` Total number of successful logins (only non-admin users) over all providers.` ?

File: services/keycloak-dashboard.adoc (hunk 2 above)
Comment (psturc) [reply services/keycloak-dashboard.adoc]: good point, will fix this

File: services/keycloak-dashboard.adoc (hunk 4 above)
Comment (psturc) [reply services/keycloak-dashboard.adoc]: Sounds better, yeah. Fixed.

**Pull request merged**